packages/widgets/src/releases/component.tsx

```diff
@@ -38,7 +45,7 @@ export default function ReleasesWidget({ options }: WidgetComponentProps<"releas
   const now = useNow();
   const formatter = useFormatter();
   const board = useRequiredBoard();
-  const [expandedRepository, setExpandedRepository] = useState({ providerKey: "", identifier: "" });
+  const [expandedRepositoryId, setExpandedRepositoryId] = useState("");
```

Comment:
```suggestion
  const [expandedRepositoryId, setExpandedRepositoryId] = useState<string | null>(null);
```
Use null instead when something is not set